projects/form-test/src/date-input/date-input.component.ts

```diff
@@ -1,27 +1,12 @@
-import { Component, forwardRef } from '@angular/core';
-import { NG_VALUE_ACCESSOR, NG_VALIDATORS, FormControl } from '@angular/forms';
-import { BaseFormAccessor } from 'projects/forms/src/lib/abstracts/base-form/base-form.accessor';
+import { Component } from '@angular/core';
+import { FormControl } from '@angular/forms';
 import { FormAccessor } from 'projects/forms/src/lib/abstracts/form/form.accessor';
+import { createAccessorProviders } from 'projects/forms/src/public-api';
```

Comment:
Klopt die import?

Reply:
Ik ga deze en de rest resolve, but for future people, deze imports zijn om ervoor te zorgen dat de test applicatie rebuild wanneer je changes maakt.

Eens we Nx hebben zijn we hier vanaf 👌